src/geometry/MergedMeshGroup.ts

```diff
@@ -199,11 +240,11 @@ export class MergedMeshGroup extends GeometryGroup {
       const mergedMesh = this.meshes[meshIndex];
 
       if (geometry == null) {
-        if (mergedMesh.geometry == null) {
+        if (this.geometry == null) {
```

Comment:
Why was this changed?

Reply:
I think having the geometry on the MergedMesh makes sense though? At least until all meshes are merged into one in this repository. The data structure here assumes an optimization that doesn't necessarily need to happen.

Reply:
It does necessarily need to happen. When I turn instanced meshes into merged meshes, I end up with a few dozen extra, tiny meshes. So I turn ALL of the meshes in a mergedMeshGroup into one object.

Reply:
Tl;dr: this lets us draw all the merged meshes in a model with 1 draw call per sector

Reply:
Talked to Anders, we're going to move everything into a single mergedMesh and delete the old meshes. Will come in a later PR, since I'm re-writing how we merge meshes anyways.